platform-db-core/src/main/java/com/softicar/platform/db/core/transaction/IDbTransaction.java

```diff
@@ -1,16 +1,61 @@
 package com.softicar.platform.db.core.transaction;
 
+import com.softicar.platform.common.core.exceptions.SofticarDeveloperException;
 import com.softicar.platform.common.core.transaction.ITransaction;
 import com.softicar.platform.common.date.DayTime;
+import com.softicar.platform.db.core.connection.IDbConnection;
 import java.util.Optional;
 import java.util.function.Supplier;
 
+/**
+ * An {@link IDbTransaction} is an object representing a logical transaction on
+ * the database management system (DBMS).
+ * <p>
+ * An {@link IDbTransaction} may represent a root (physical) transaction or a
+ * nested (logical) transaction.
```

Comment:
The definition of `logical` seems to differ from that in the sentence before. I suggest "simulated" or "emulated" here.

Reply:
changed text